src/Saas.Admin/readme.md

```diff
@@ -8,70 +8,74 @@ The SaaS Admin Service is an API that is reponsible for tenant management operat
+| AzureAdB2C:SignUpSignInPolicyId            | Name of signup/signin policy                                                                                                            | false  | B2C_1A_SIGNUP_SIGNIN                  |
+| AzureAdB2C:TenantId                        | Identifier for the overall Azure AD B2C tenant for the overall SaaS ecosystem                                                           | true   |                                       |
+| ClaimToRoleTransformer:AuthenticationType  | Indicates the Authentication type for new identity                                                                                      | false  | MyCustomRoleAuth                      |
+| ClaimToRoleTransformer:RoleClaimtype       | Type of the claim to use in the new Identity, works alongside built-in                                                                  | false  | MyCustomRoles                         |
+| ClaimToRoleTransformer:SourceClaimType     | Name of the claim custom roles are in                                                                                                   | false  | permissions                           |
+| ConnectionStrings:TenantsContext           | Connection String to SQL server database used to store permission data.                                                                 | true   |                                       |
+| KeyVault:PermissionsApiCert                | The name of the secret in Azure Key Vault that contains a base64 encoded certificate to use for authentication with the permissions api | false  |                                       |
+| KeyVault:Url                               | KeyVault URL to pull secret values from in production                                                                                   | false  |                                       |
+| Logging:LogLevel:Default                   | Logging level when no configured provider is matched                                                                                    | false  | Information                           |
+| Logging:LogLevel:Microsoft.AspNetCore      | Logging level for AspNetCore logging                                                                                                    | false  | Warning                               |
+| PermissionsApi:BaseUrl                     | URL for downstream [Permissions API](../Saas.Identity/Saas.Permissions/readme.md)                                                       | false  |                                       |
+| PermissionsApi:LocalCertificate            | A Base64 encoded certificate (.CER) used to authenticate with the permissions API. Only used for local development.                     | true   |                                       |
```

Comment:
Add an additional one here, please:

`PermissionsApi:PermissionsApiCertPassphrase`
`The passphrase used to protect the certificate file during creation.`
`secret=true`